### ISSUE TITLE
Add to README: How is the .ISO built?

### ISSUE THREAD

**Issue by ghost** [DonyaOS/Donya#3]
Hiya!
Since this is an experimental project (yet simple in this state), and there would be many (such as myself) that would be interested in figuring out how Donya works, I'd like to see an overview of the build process and structure in layman's terms put somewhere into the `README`. Personally speaking, I've been super interested in working on a distro, and would love to contribute to Donya, if I knew how it worked.
My deepest exposure to Linux has been using Ubuntu as my main OS for about a year now, and my skills as a software developer only really extend to Java, C#, JavaScript, and D & Vala to a far lesser extent, so I'm still really bad when it comes to computers, haha. Sorry if this seems like a stupid request but I'm struggling to learn about Linux & FOSS and I think getting a chance to understand Donya would be educational for me personally, and possibly others, thus leading to more contributors on the project.

**Comment by BaseMax** [comment]
Hi Emil,

Thank you for your message.

At current time, main bash script is [donya.sh](https://github.com/DonyaOS/Donya/blob/815216b91e7df81c9d3aebc1217cd9a2b3d698b3/donya.sh)

At first, you need to download Linux kernel, BusyBox, syslinux...
You can download it manually or remove '#' in the script file.

```
# wget https://kernel.org/pub/linux/kernel/v5.x/linux-5.8.tar.xz
# wget http://busybox.net/downloads/busybox-1.32.0.tar.bz2
# wget http://kernel.org/pub/linux/utils/boot/syslinux/syslinux-6.03.tar.xz
```

Then continue running `$ bash donya.sh`

One of my friends tried this. So this should work. @esmaeelE 

Tell me, if we can help with anything else at the moment.

I will be happy if you also participate and help in the development.

Regards,

**Comment by ghost** [comment]
Thanks for the response! I'll close this issue now. You're really helpful; best of luck to both you and the Donya project.
